packages/amplify/amplify_flutter/lib/src/hybrid_impl.dart

```diff
@@ -80,6 +80,8 @@ class AmplifyHybridImpl extends AmplifyClass {
               AmplifyExceptionMessages.missingRecoverySuggestion,
         );
       }
+    } on AmplifyAlreadyConfiguredException {
+      return;
```

Comment:
Maybe we want to set the isConfigured flag to true here?

Reply:
It's set by completing asyncConfig. If we hot restart these will throw but by ignoring them, asyncConfig will be set